Rookies/README.md

```diff
@@ -19,6 +19,17 @@ You can read the [Heroes Review Criteria and Scoring Matrix Guidelines](../Heroe
 - Personal Site: [https://www.parity.io/](https://www.parity.io/)
 - Blog: [https://www.parity.io/blog/](https://www.parity.io/blog/)
 
+
+## Fenn Ignatius Saji
```

Comment:
If you could just change this to a H3: `###` that would be great:

```suggestion
### Fenn Ignatius Saji
```

Reply:
> If you could just change this to a H3: `###` that would be great:

Updated